.gitignore

```diff
@@ -6,6 +6,7 @@
 
 # Installation artifacts
 src
+requirements/*.txt
```

Comment:
Yes, these are generated by pip-tools through the make command:

    make upgrade

The idea is that it is easy to pin the requirements but also easy to refresh them all as needed:

https://github.com/nvie/pip-tools